modules/persistent-storage-csi-gcp-hyperdisk-limitations.adoc

```diff
@@ -0,0 +1,20 @@
+// Module included in the following assemblies:
+//
+// * storage/container_storage_interface/persistent-storage-csi-gcp-pd.adoc
+
+:_mod-docs-content-type: CONCEPT
+[id="persistent-storage-csi-gcp-hyperdisk-limitations_{context}"]
+= C3 and N4 instance type limitations
+The GCP PD CSI driver support for the C3 instance type for bare metal and N4 machine series have the following limitations:
+
+* Hyperdisk-balanced disks with RWX is not supported.
+
+* Storage pool cloning is not supported.
+
+* For cloning or resizing, hyperdisk-balanced disks original volume size must be 6Gi or greater.
+
+* A GCP cluster with n4-standard-2 or n4-standard-4 nodes can erroneously exceed the maximum attachable disk number, which should be 16.
+
+* The default storage class is standard-csi.
```

Comment:
I think we should rewrite this to point to the section "Creating the storage class for C3 and N4 instance types"

Reply:
Did the best I can there. We have some limitations regarding xrefs in our tool.

Reply:
fair, can we be more prescriptive then ? Somelike like "you need to create a SC manually"

Reply:
Done.